<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientForm</class>
 <widget class="QWidget" name="ClientForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mi Deporte Movil</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/ico2.ico</normaloff>:/img/ico2.ico</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>511</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-2</number>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color:rgba(0,0,0,0.7);

selection-color: white;  
selection-background-color:rgba(0,0,0,0.5);     

border-radius: 10px;

</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>   Reservas  </string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>420</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Myungjo Std M</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton {
	color: #333;
	border: 2px solid #555;
	border-radius: 11px;
	padding: 5px;
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #888);
	min-width: 80px;
}
#pushButton:hover {
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}
#pushButton:pressed {
	background: qradialgradient(cx: 0.4, cy: -0.1,
	fx: 0.4, fy: -0.1,
	radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}</string>
     </property>
     <property name="text">
      <string>Reservar</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbSports">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color:rgba(0,0,0,0.7);
selection-color: white;  
selection-background-color:rgba(0,0,0,0.5);     
border: 2px groove gray;
border-radius: 10px;
padding: 2px 4px;
</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbClubs">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color:rgba(0,0,0,0.7);
selection-color: white;  
selection-background-color:rgba(0,0,0,0.5);     
border: 2px groove gray;
border-radius: 10px;
padding: 2px 4px;
</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbGameFields">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color:rgba(0,0,0,0.7);
selection-color: white;  
selection-background-color:rgba(0,0,0,0.5);     
border: 2px groove gray;
border-radius: 10px;
padding: 2px 4px;
</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0.7);
color:white;</string>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCalendarWidget QAbstractItemView
{
background-color: rgb(192,192,192); /* цвет фона текущего месяца */
selection-background-color: black; /* цвет фона выбранного дня */
selection-color: white; /* цвет текста выбранного дня */
}
QCalendarWidget QMenu {
  	color: white;
  	background-color: rgb(100, 100, 100);
	selection-background-color: black; /* цвет фона выбранного дня */
	selection-color: white; /* цвет текста выбранного дня */
  }

QCalendarWidget QMenu::hover{
	color: rgb(158, 158, 158);
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-30</y>
       <width>350</width>
       <height>521</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img.qrc">:/img/background.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_6</zorder>
    <zorder>pushButton</zorder>
    <zorder>cmbSports</zorder>
    <zorder>cmbClubs</zorder>
    <zorder>cmbGameFields</zorder>
    <zorder>list</zorder>
    <zorder>calendarWidget</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>   Anuncios   </string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>341</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="title">
      <string>Ultimas Publicaciones</string>
     </property>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>321</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0.7);
color:white;</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbAnuncio">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>341</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="title">
      <string>Crear Publicacion</string>
     </property>
     <widget class="QTextEdit" name="teMessage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>321</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0.7);
color:white;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbPublic">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Myungjo Std M</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#pbPublic {
	color: #333;
	border: 2px solid #555;
	border-radius: 11px;
	padding: 5px;
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #888);
	min-width: 80px;
}
#pbPublic:hover {
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}
#pbPublic:pressed {
	background: qradialgradient(cx: 0.4, cy: -0.1,
	fx: 0.4, fy: -0.1,
	radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}</string>
      </property>
      <property name="text">
       <string>Publicar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbClear">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>120</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Myungjo Std M</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#pbClear {
	color: #333;
	border: 2px solid #555;
	border-radius: 11px;
	padding: 5px;
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #888);
	min-width: 80px;
}
#pbClear:hover {
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}
#pbClear:pressed {
	background: qradialgradient(cx: 0.4, cy: -0.1,
	fx: 0.4, fy: -0.1,
	radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}</string>
      </property>
      <property name="text">
       <string>Borrar</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>350</width>
       <height>511</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img.qrc">:/img/background.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_7</zorder>
    <zorder>groupBox</zorder>
    <zorder>gbAnuncio</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string> Mis Reservas </string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>341</width>
       <height>481</height>
      </rect>
     </property>
     <property name="title">
      <string>Historial</string>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-30</y>
        <width>350</width>
        <height>521</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="img.qrc">:/img/background.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="listMyReservations">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>321</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0.7);
color:white;</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listReservationHours">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>321</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0.7);
color:white;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>   Mis Datos   </string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-20</y>
       <width>350</width>
       <height>511</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img.qrc">:/img/background.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>341</width>
       <height>471</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="title">
      <string>Mis Datos</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Myungjo Std M</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
background-color:none;</string>
      </property>
      <property name="text">
       <string>E-mail:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Myungjo Std M</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
background-color:none;</string>
      </property>
      <property name="text">
       <string>Telefono:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Myungjo Std M</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
background-color:none;</string>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Myungjo Std M</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
background-color:none;</string>
      </property>
      <property name="text">
       <string>Apellido:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lName">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Jokerman</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font: 15pt &quot;Jokerman&quot;;
background-color:rgba(0,0,0,0.7);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lLast">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Jokerman</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font: 15pt &quot;Jokerman&quot;;
background-color:rgba(0,0,0,0.7);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lPhone">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Jokerman</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font: 15pt &quot;Jokerman&quot;;
background-color:rgba(0,0,0,0.7);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lEmail">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>250</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Jokerman</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font: 15pt &quot;Jokerman&quot;;
background-color:rgba(0,0,0,0.7);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbCloseSession">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pbCloseSession {
	color: #333;
	border: 2px solid #555;
	border-radius: 11px;
	padding: 5px;
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #888);
	min-width: 80px;
}
#pbCloseSession:hover {
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}
#pbCloseSession:pressed {
	background: qradialgradient(cx: 0.4, cy: -0.1,
	fx: 0.4, fy: -0.1,
	radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}</string>
   </property>
   <property name="text">
    <string>Cerrar Sesion</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
